cmd/metrics-v2.go

```diff
@@ -534,7 +541,61 @@ func getS3TTFBDistributionMD() MetricDescription {
 		Type:      gaugeMetric,
 	}
 }
+func getMinioFDOpenMD() MetricDescription {
+	return MetricDescription{
+		Namespace: minIOProcessNamespace,
+		Subsystem: fileDescriptorSubsystem,
+		Name:      openTotal,
+		Help:      "Total number of open file descriptors by the MinIO Server process.",
+		Type:      gaugeMetric,
+	}
+}
 
+func getMinioFDLimitMD() MetricDescription {
+	return MetricDescription{
+		Namespace: minIOProcessNamespace,
+		Subsystem: fileDescriptorSubsystem,
+		Name:      limitTotal,
+		Help:      "Limit on total number of open file descriptors for the MinIO Server process.",
+		Type:      gaugeMetric,
+	}
+}
+
+func getMinioFDMetrics() MetricsGroup {
+	return MetricsGroup{
+		Metrics: []Metric{},
+		initialize: func(ctx context.Context, metrics *MetricsGroup) {
+			pid := os.Getpid()
+			p, err := procfs.NewProc(pid)
+			if err != nil {
+				logger.LogOnceIf(ctx, err, minIOProcessNamespace)
```

Comment:
The way to do this I see is `proc.Self()` @kerneltime

Reply:
Cool!